### ISSUE TITLE
Pass intrinsic calibration as prior information

### ISSUE THREAD

**Issue by tompollok** [colmap/colmap#1033]
Is there a way to pass colmap an intrinsic calibration for specific images and even maybe prevent optimization during BA?

**Comment by tsattler** [comment]
Yes, you can specify intrinsics both from the command line and in the gui. You can also fix the intrinsic parameters during the reconstruction. However, this will fix all intrinsics.

**Comment by DaniilSNikulin** [comment]
Colmap already have parameter `--ImageReader.single_camera_per_folder`.
In private fork I have parameter `--camera_params_per_folder_path` which specify mapping 'folder name -> intrinsics'.
I could provide PR with this parameter if COLMAP owners (@ahojnnes) will be agree.

**Comment by ahojnnes** [comment]
This would be a useful feature. We should also document it appropriately, if we add it. Thanks!

**Comment by tompollok** [comment]
That sounds great!

**Comment by tompollok** [comment]
Btw is anywhere an evaluation of how sfm / BA based intrinsic calibration compared to classical chessboard approaches? I guess it depends on the number of overlapping images and correspondences found. Did anybody look into this in the past?

**Comment by tompollok** [comment]
@DaniilSNikulin any plans to make a PR anytime soon?
